New version: OnlineStatsChains v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: OnlineStatsChains
- Repository: https://github.com/femtotrader/OnlineStatsChains.jl
- Created by: @femtotrader
- Version: v0.3.0
- Commit: 7886a0a39419e26f7197162b012382db4400abd5
- Reviewed by: @femtotrader
- Reference: https://github.com/femtotrader/OnlineStatsChains.jl/issues/2
- Description: A Julia package for chaining OnlineStats computations in a Directed Acyclic Graph (DAG) structure with automatic value propagation
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- add Aqua QA and DeepWiki badges to README
- add Aqua.jl quality assurance requirements to core
- update documentation and CI configuration
- feature: implement observer system for Rocket.jl integration
- refactor: split OnlineStatsChains.jl into modular components
- docs: reorganize project guides into guides/ directory
- docs: add GitHub repository link to documentation
- feat: add edge transformers
- feat: add filtered edges support for conditional propagation
- docs: add LICENSE file and release readiness documentation
- ci: initial setup
- docs: add comprehensive setup guide
- docs: add conventional commits and pre-commit hooks requirements
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598d99d30b1f4e1b1702fcc6909610cc8efb43a9e6f89e005779227925f10389375643904c50a5086e80ba143dd1acb677bf5f248d0af62e2ef94c1bf6502aa5f75c0ed1c7e147b40e4911f81f77eac6ef70bca8061e1890b99324c02424396b1faeeb41929759a331648b4d3c30ef538eca217ebb2606479b96da1c1b85bbe210e -->

femtotrader commented:
https://github.com/JuliaRegistries/General/pull/139646 ever register.
This one should be stopped

JuliaRegistrator committed:
New version: OnlineStatsChains v0.3.0

UUID: c3092dcd-a09e-45f3-b86d-18109bff285f
Repo: https://github.com/femtotrader/OnlineStatsChains.jl.git
Tree: 0f65641650573d7fd2066630c44298b656634620

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

femtotrader commented:
[merge approved]
